CMakeLists.txt

```diff
@@ -98,6 +98,10 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   add_compile_definitions(MAC_OSX)
 endif()
 
+if(CMAKE_CROSSCOMPILING AND DEPENDS_ALLOW_HOST_PACKAGES)
```

Comment:
Can't this be done in the toolchain file by conditionally setting the `CMAKE_FIND_ROOT_PATH_*` vars?

Reply:
That was my initial idea as well :)

It turns out that a toolchain file is being processed more than once (the actual number of runs depends on CMake version). Therefore, its processing must be idempotent. `list(APPEND ...)` command is not idempotent. That is why it has been moved out of the toolchain file.